wai-extra/Network/Wai/Middleware/RequestLogger.hs

```diff
@@ -81,12 +84,19 @@ data RequestLoggerSettings = RequestLoggerSettings
     , destination :: Destination
     }
 
+-- | @defaultRequestLoggerSettings@ is the original 'Default' instance
+--   factored out into a top-level declared variable.
+--   @since 3.0.22.2
```

Comment:
This is just from memory so could be wrong, but does the @since need to be separated by an empty comment to work?